checker/src/org/checkerframework/checker/index/upperbound/OffsetEquation.java

```diff
@@ -420,9 +420,9 @@ public static OffsetEquation createOffsetFromNodesValue(
         if (node.getTree() != null && TreeUtils.isExpressionTree(node.getTree())) {
             Long i;
             if (op == '+') {
-                i = IndexUtil.getMinValue(node.getTree(), factory.getValueAnnotatedTypeFactory());
+                i = IndexUtil.getExactValue(node.getTree(), factory.getValueAnnotatedTypeFactory());
```

Comment:
Yes. This was causing some problems. I think we really only want the exact value here, too.